addon/templates/components/polaris-form.hbs

```diff
@@ -0,0 +1,7 @@
+{{yield}}
+
+{{#if implicitSubmit}}
+  {{#polaris-visually-hidden}}
+    <button type="submit" aria-hidden="true" data-test-form-button />
```

Comment:
I seem to recall self-closing elements like this causing linter complaints or something... can't remember where/when though 😞

Reply:
will just write it standard way...copy-pasta fail....though linter didn't complain